treehouses/remote
# status: fast loading

### Testing:

1. Upgrade CLI to "statusjason" branch by running the command `treehouses upgrade cli statusjason`
2. Run the remote on this "fast-status" branch.
3. Go to the Status page, everything should load simultaneously except cli upgrade check.
